Commands.md

```diff
@@ -196,7 +196,7 @@ project  : git-extras
 
 ## git line-summary
 
-  WARNING: git line-summary has been replaced by git summary --line and will be removed in a future release. 
+  WARNING: git line-summary has been replaced by git summary --line and will be removed in a future release.
```

Comment:
Oops, looks like atom removed a load of unnecessary white space when I saved. Happy to split this out into a separate commit (or just remove it) if you want.